readthedocs/api/base.py

```diff
@@ -32,7 +32,7 @@
 class ProjectResource(ModelResource, SearchMixin):
     users = fields.ToManyField('readthedocs.api.base.UserResource', 'users')
 
-    class Meta:
+    class Meta(object):
```

Comment:
This feels un-djangoy -- have they suggested doing this now, or is this just to make the linter happy?